ts/Dashboards/Plugins/HighchartsTypes.ts

```diff
@@ -0,0 +1,251 @@
+/* *
+ *
+ *  (c) 2009-2023 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Sophie Bremer
+ *
+ * */
+
+
+'use strict';
+
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+
+import type ColorString from '../../Core/Color/ColorString';
+
+
+/* *
+ *
+ *  Declarations
+ *
+ * */
+
+
+export interface AnimationOptions {
+}
+
+export interface Axis {
```

Comment:
Can't we get the reference to these types instead of writing them once again?

Reply:
That is essential to keep the Highcharts Core out. We need the API definition for Plugin implementation, but not the types in the distribution, except Options. It would be easier to keep it out, if we had our own repository, because then we could reference highcharts as a package.

Reply:
Now thinking about we could simulate it with the tsconfig. It is just a question how this then maps to the classic bundle.

Reply:
No luck with tsconfig. 😞 We might change the repository structure and the we can try ones more.

Reply:
let's leave it as it is now ;) thanks!